core/drivers/gic.c

```diff
@@ -996,6 +1024,26 @@ static void gic_op_raise_sgi(struct itr_chip *chip, size_t it,
 	gic_it_raise_sgi(gd, it, cpu_mask, ns);
 }
 
+static void gic_it_reset(struct gic_data *gd, size_t it)
```

Comment:
this whole function `gic_it_reset()` seems supported only when `CFG_ARM_GICV3` is enabled.

Reply:
It also works when CFG_ARM_GICV3 is not enabled, The GIC version affects whether IGROUPMODR is configured.